src/vs/workbench/browser/parts/editor/breadcrumbs.ts

```diff
@@ -35,7 +35,7 @@ export class BreadcrumbsService implements IBreadcrumbsService {
 
 	register(group: number, widget: BreadcrumbsWidget): IDisposable {
 		if (this._map.has(group)) {
-			throw new Error(`group (${group}) has already a widget`);
+			console.error(`group (${group}) has already a widget`);
```

Comment:
What is the purpose of this change?

Reply:
Same as https://github.com/microsoft/vscode/pull/179507#discussion_r1170801938

Reply:
I think this would not be needed if we can avoid the hack for restoring the editors.